src/main/java/com/minecraftabnormals/abnormals_core/core/util/BiomeUtil.java

```diff
@@ -0,0 +1,35 @@
+package com.minecraftabnormals.abnormals_core.core.util;
+
+import com.mojang.datafixers.util.Pair;
+import net.minecraft.util.RegistryKey;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.WeightedList;
+import net.minecraft.world.biome.Biome;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class BiomeUtil {
+	private static final Map<RegistryKey<Biome>, WeightedList<RegistryKey<Biome>>> hillBiomeMap = new HashMap<>();
+
+	public static void addHillBiome(RegistryKey<Biome> biome, Pair<RegistryKey<Biome>, Integer>... hills) {
+		for (Pair<RegistryKey<Biome>, Integer> hill : hills)
```

Comment:
`biome` remains constant in this method so it doesn't make a lot of sense to do `computeIfAbsent` over and over in a loop.